### PR TITLE
Fix readthedoc to compile properly by installing Cython

### PR DESCRIPTION
**Pull request by yuyu2172**

### CODE REVIEW

**Comment by Hakuyume** [comment]
In the title of the PR, `copile` -> `compile`?

**Comment by yuyu2172** [comment]
Thanks

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-chainer
-scikit-image
-scikit-learn
+chainer=1.24
+Cython
 matplotlib
+numpy
+pillow
```
Comment (Hakuyume) [inline docs/requirements.txt]: Both `Cython` and `Pillow` start with an upper character in pypi. Why are you using upper character only for `Cython`?

Comment (yuyu2172) [inline docs/requirements.txt]: Sorry, I didn't know that Pillow start with a capital letter.

Comment (Hakuyume) [inline docs/requirements.txt]: https://pypi.python.org/pypi/Pillow/4.1.1

**Comment by Hakuyume** [comment]
LGTM

**Pull request merged**